crates/markdown_preview/src/markdown_preview_view.rs

```diff
@@ -130,7 +130,7 @@ impl MarkdownPreviewView {
             editor,
             workspace_handle,
             language_registry,
-            "Markdown Preview".into(),
+            "".into(),
```

Comment:
Let's change this a bit:

* `""` is a bit too magical, what we can do is use `tab_content_text: Option<SharedString>` as a default value.

* then, inside `fn tab_content_text` it would be something like `self.tab_content_text.cloned().unwrap_or_else(|| SharedString::from("Markdown Preview"))`.

* and the `if self.tab_content_text.is_empty()` check will do `is_none` instead.